src/wallet/rpcwallet.cpp

```diff
@@ -2369,6 +2369,7 @@ static UniValue getbalances(const JSONRPCRequest& request)
         RPCResult{
             RPCResult::Type::OBJ, "", "",
             {
+                {RPCResult::Type::STR_HEX, "lastblock", "The hash of the block the balances were valid on"},
```

Comment:
Could move this last, since it is the least-important information?

Reply:
Done.